examples/inverse/plot_custom_inverse_solver.py

```diff
@@ -40,8 +40,8 @@
 
 evoked = evoked.pick_types(eeg=False, meg=True)
 # Handling forward solution
-forward = mne.read_forward_solution(fwd_fname, surf_ori=True)
-
+forward = mne.read_forward_solution(fwd_fname)
+forward = mne.convert_forward_solution(forward, surf_ori=True)
```

Comment:
We could, but there is no change, when we change the default from None to True in this case. Should we add use_cps=True in each of these calls even if the behavior will not change when we change the default?